pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/documentdb/index.ts

```diff
@@ -5,10 +5,12 @@ import * as pulumi from "@pulumi/pulumi";
 import * as utilities from "../utilities";
 
 // Export members:
-export * from "./sqlResourceSqlContainer";
```

Comment:
So this is a typical change example. Instead of reexporting `*` which generates `require()` at runtime we reexport equivalent definitions below.